guides/screen-reader-differences.md

```diff
@@ -0,0 +1,126 @@
+
+They recommend testing the following pairings when designing and developing websites:
+
+- Firefox with NVDA
+- Internet Explorer with JAWS
+- Safari with VoiceOver
+- Edge with Narrator
+
+These also reflect the [most popular browser/screen reader combinations](https://webaim.org/projects/screenreadersurvey7/#browsercombos). An important consideration when conducting our own screen reader testing is that **10% of users surveyed use VoiceOver with Safari, and only 1.4% use VoiceOver with Chrome**.
+
+### Interaction modes
+
+In addition to reading the page content in a linear order, screen readers also offer modes for users to interact with specific types of webpage content. These vary across the different applications.
+
+#### 1. Read/Browse mode
+This mode is available in Windows screen readers like JAWS and NVDA, **and** in Mac OS and iOS VoiceOver. For VoiceOver, this mode is triggered using `control + option + A`.
+
+Keyboard shortcuts can navigate through specific items in the page in this mode. For example, on VoiceOver you can type `control + option + command + H` to navigate through all the headings on the page. In JAWS, the user can do the same thing with just the `H` key.
+
+#### 2. VoiceOver rotor
+
+This mode is only available in VoiceOver. The rotor provides a menu interface for interacting with specific types of content that is present on the current webpage. 
+
+The user can navigate between menus in the rotor using the left and right arrow keys. The up and down arrow keys navigate through the list of elements in a particular menu. For example, there is a Headings menu which contains all of the **currently visible** headings that are on the page in a list, like this:
+
+
+Headings
+- 1: H1
+- 2: H2
+- 3: H3
+- 3: H3
+
+![screen shot 2018-06-28 at 11 31 26 am](https://user-images.githubusercontent.com/704760/42045575-d944241c-7ac9-11e8-9473-ce2b2726cac7.png)
+
+
+
+Because hidden elements are not considered part of the page content for screen readers*, they are not included in the elements listed by the rotor menus. This has important implications for how we design and code interactive elements on our webpages. For example, in the current implementation of our global navigation "mega menu", none of the content that's in the mega menu dropdown is accessible to the VoiceOver rotor unless the user has opened the dropdown with a mouse hover or the keyboard. This as a general problem with drop down menus and similar interaction designs that affects VoiceOver users (and likely all screen reader users- we just haven't tested other software yet).
+
+*This is not the case for elements that are visually hidden using the same CSS techniques as are used to visually hide the "Skip nav" link on every page. It only applies to techniques that hide the content in a straightforward manner such as using `display: none` or `visibility: hidden`.
```

Comment:
Using an asterisk for a note will cause the text that follows to be rendered as italic. I typically wrap the note in underscores to make it italic and prepend the text with `NOTE:`